src/alarms/notifications/AlarmRow.js

```diff
@@ -100,14 +42,27 @@ class AlarmRow extends Component {
           </div>
 
           <div>
-            <NavLink
-              to={`/alarms/notifications/${alarm.uuid}`}
-              style={{
-                color: "#333"
-              }}
-            >
-              {alarm.name}
-            </NavLink>
+            {alarm.url.includes("timeseriesalarms") ? (
+              <span
+                to={`/alarms/notifications/${alarm.uuid}`}
+                style={{
+                  color: "#333",
+                  cursor: "auto"
+                }}
+              >
+                {alarm.name}
+              </span>
+            ) : (
+              <NavLink
+                to={`/alarms/notifications/${alarm.uuid}`}
+                style={{
+                  color: "#333"
+                }}
+              >
+                {alarm.name}
+              </NavLink>
+            )}
+
```

Comment:
Is het verschil tussen beide uitkomsten van de ? alleen cursor: "auto"?

Reply:
Behalve het is de ene een navlink en de ander een span.

Reply:
@remcogerlich 
Zullen wij nog naar deze branch kijken?